### PR TITLE
add core examples back to index.html

### PR DESCRIPTION
**Pull request by ngokevin**

### CODE REVIEW

File: examples/index.html [hunk 1]
```diff
@@ -174,17 +164,30 @@
...
     </ul>
 
-    <h2>Components</h2>
-
-    <ul class="links">
-      <li><a href="entitycomponent-basics/">Basics</a></li>
-    </ul>
-
-
     <h2>Tests</h2>
```
Comment (cvan) [inline examples/index.html]: is "Tests" the correct heading?

File: examples/index.html [hunk 2]
```diff
@@ -174,17 +164,30 @@
...
     <ul class="links">
-      <li><a href="animation-template-test/">Animations + Templates</a></li>
+      <li><a href="animation/">Animation</a></li>
+      <li><a href="controls/">Controls</a></li>
+      <li><a href="cursor/">Cursor</a></li>
+      <li><a href="cube/">Cube</a></li>
+      <li><a href="cylinders/">Cylinders</a></li>
+      <li><a href="fog/">Fog</a></li>
+      <li><a href="geometries/">Geometries</a></li>
+      <li><a href="geometry-gallery">Geometry Gallery</a></li>
```
Comment (cvan) [inline examples/index.html]: `/`

File: examples/index.html (hunk 1 above)
Comment (ngokevin) [inline examples/index.html]: It's sufficient I think. We use it to test so they're test examples.

Comment (cvan) [inline examples/index.html]: okay, thought that was an accident. that's fine, yeah

**Comment by dmarcos** [comment]
We have to change all the paths of the examples. They still point to ```<script src="../../dist/aframe-core.js"></script>``

**Comment by ngokevin** [comment]
Ah, weird they were still working. Must be Budo magic.

**Pull request merged**